cmd/crd-schema-gen/generator/generator.go

```diff
@@ -66,11 +85,16 @@ func Run() error {
 		fmt.Print(string(out))
 		return err
 	}
+	if err := ioutil.WriteFile(filepath.Join(tmpDir, "PROJECT"), []byte(fmt.Sprintf(`
```

Comment:
If we merge https://github.com/openshift/kubernetes-sigs-controller-tools/pull/6 you shouldn't need this, right?

Reply:
Rather, you'd specify them when instantiating the generator

Reply:
yeah